modules/labs-react/common/lib/storybook-utils/ComponentStatesTable.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {styled} from '@workday/canvas-kit-react/common';
+
+/**
+ * A helper to generate a table of all possible states for component visual testing.
+ */
+export interface ComponentStatesTableProps {
+  /**
+   * The props that will be passed to the component when it is rendered and the corresponding
+   * row label for that permutation. It is encouraged to use the result of permutateProps()
+   * rather than passing in a list so we don't miss any combinations.
+   */
+  rowProps: PropCombination[];
+  /**
+   * The props that will be passed to the component when it is rendered and the corresponding
+   * column label for that permutation. It is encouraged to use the result of permutateProps()
+   * rather than passing in a list so we don't miss any combinations.
+   */
+  columnProps: PropCombination[];
+  /**
+   * The render function called to render the component in each cell of the table. This gives you
+   * the ability to add extra styling or markup (a blue background for an inverse variant, for example).
+   */
+  children(props: Props): React.ReactNode;
+}
+
+const Table = styled('table')({
+  width: '100%',
+  thead: {
+    textAlign: 'left',
+    paddingBottom: 16,
+  },
+  'td, th': {
+    minWidth: 100,
+    paddingBottom: 16,
+    paddingRight: 16,
+    textAlign: 'left',
+  },
+});
+
+export const ComponentStatesTable = ({
+  rowProps,
+  columnProps,
+  children,
+}: ComponentStatesTableProps) => {
```

Comment:
Changed this to a FC from a class. Also dropped the default props, I couldn't find any module actually using them, so figured it was better to just make them required.